types/swagger-schema-official/index.d.ts

```diff
@@ -129,11 +150,12 @@ export interface Response {
 }
 
 // ------------------------------ Schema -------------------------------------
-export interface BaseSchema {
+export type BaseSchema = {
+  type?: ParameterType;
   format?: string;
   title?: string;
   description?: string;
-  default?: string | boolean | number | {};
+  default?: any[] | boolean | number | null | object | string;
```

Comment:
is it `any` at this point?

Reply:
Hmm I'm against any in general, I would say I was too generous and I should check it out again and narrow it down, although it can really be anything here. I'll check it out again.